src/app/containers/NewStoryPromo/PromoAtoms/Heading/index.styles.jsx

```diff
@@ -0,0 +1,11 @@
+import { GEL_SPACING } from '#app/legacy/gel-foundations/src/spacings';
+import { C_GREY_10 } from '#app/legacy/psammead-styles/src/colours';
+import { getSerifMedium } from '#app/legacy/psammead-styles/src/font-styles';
+import styled from '@emotion/styled';
+
+export default styled.h3`
+  color: ${C_GREY_10};
+  margin: 0; /* Reset */
+  padding-bottom: ${GEL_SPACING};
+  ${({ service }) => getSerifMedium(service)}
```

Comment:
Not sure it's practical to resolve this in this PR but just want to point out we want to move away from pulling in reusable css via functions and instead use base components: https://paper.dropbox.com/doc/Draft-component-standards--Bkx5rEOyAQjcAuS~y7A3j9eEAg-vm3tnpOZr4MxtFrBzElmR#:uid=826389717039756463484127&h2=CSS-in-JS:-For-style-reusabili

Reply:
Not sure if we want to do this now. For now, I will skip this implementation that you have suggested. But if more time allows I will have a deep dive into this.

Reply:
yeah don't do it now, I'm just pointing it out for awareness, I think we should come back to this area a few times and try and apply our new approaches but not now.